src/main/java/io/rocketpartners/cloud/action/sql/SqlDb.java

```diff
@@ -317,19 +320,19 @@ public void delete(Table table, List<String> entityKeys) throws Exception
          String sql = "";
          sql += " DELETE FROM " + quoteCol(table.getName());
```

Comment:
Are you using the Rocket Partners eclipse code formatter xml?  We seem to be getting DIFF differences that are just formatter based. https://github.com/RocketPartners/rocket-java/blob/release-2.0.x/utils/Eclipse%20Code%20Formatter.xml

Reply:
Yes, I'm using the same Rocket Partners eclipse code formatter xml